solana/programs/example-native-token-transfers/src/instructions/admin.rs

```diff
@@ -73,6 +79,28 @@ pub fn transfer_ownership(ctx: Context<TransferOwnership>) -> Result<()> {
     )
 }
 
+pub fn transfer_ownership_one_step(ctx: Context<TransferOwnership>) -> Result<()> {
```

Comment:
Any thoughts on renaming this to something like `transfer_ownership_unchecked` or `transfer_ownership_one_step_unchecked` (or `dangerous` instead of unchecked etc.)?

It might be nice to surface the `_unchecked` naming from the BPF loader call into our API so that users know exactly what they're up to and are encouraged to read the documentation thoroughly.

Reply:
funnily enough I originally named it exactly `transfer_ownership_unchecked` but wasn't satisfied. I like `transfer_ownership_one_step_unchecked`. It's a mouthful, but that's a feature in this case